template.md

```diff
@@ -0,0 +1,305 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: [MANAGER ALIAS]
```

Comment:
It seems all articles in core-docs currently have `manager: wpickett`. Should that be hardcoded here in the template?

Especially since I would expect people to be confused by what to write here.